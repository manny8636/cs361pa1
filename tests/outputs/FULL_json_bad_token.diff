--- outputs/FULL_json_bad_token.txt	2022-02-13 13:37:52.071601782 -0500
+++ expected/FULL_json_bad_token.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,2 @@
-Success!
+SYNTAX ERROR: Could not process text beginning at 'not : "good" }'
+Parsing ../data/json_bad_token.txt failed
