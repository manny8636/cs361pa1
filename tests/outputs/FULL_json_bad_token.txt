Parsing ../data/json_bad_token.txt failed
