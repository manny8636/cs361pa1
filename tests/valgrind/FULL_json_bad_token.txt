==372166== Memcheck, a memory error detector
==372166== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372166== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372166== Command: ../json -o ../data/json_bad_token.txt
==372166== 
==372166== 
==372166== HEAP SUMMARY:
==372166==     in use at exit: 2,064 bytes in 1 blocks
==372166==   total heap usage: 4 allocs, 3 frees, 10,848 bytes allocated
==372166== 
==372166== LEAK SUMMARY:
==372166==    definitely lost: 2,064 bytes in 1 blocks
==372166==    indirectly lost: 0 bytes in 0 blocks
==372166==      possibly lost: 0 bytes in 0 blocks
==372166==    still reachable: 0 bytes in 0 blocks
==372166==         suppressed: 0 bytes in 0 blocks
==372166== Rerun with --leak-check=full to see details of leaked memory
==372166== 
==372166== For lists of detected and suppressed errors, rerun with: -s
==372166== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
