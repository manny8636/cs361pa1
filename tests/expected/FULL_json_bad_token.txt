SYNTAX ERROR: Could not process text beginning at 'not : "good" }'
Parsing ../data/json_bad_token.txt failed
